### DISCUSSION TITLE
CellComponentProps is not working when passing copied: false

### DISCUSSION THREAD

**Discussion by Milan-Koladiya** [iddan/react-spreadsheet#361] [Q&A]
**Goal:- my goal is to prevent cells from copy**

I have tried CellComponentProps and passed copied: false but still it copies. did I use it in the wrong way? please check the below code.

` <Spreadsheet
                CellComponentProps={(cell) => ({
                  copied: false, 
                  ...cell,
                })}
                onChange={(updatedData) => setUpdatedGridData({ ...updatedGridData, income: updatedData })}
                data={gridData.income}
                columnLabels={columnHeaders}
                rowLabels={incomeGridTitle}
              />`

**Comment by iddan** [comment]
The component currently doesn't support disabling copy a cell

**Reply by Milan-Koladiya** [reply]
okay, I see it in the react-spreadsheet [official documentation](https://iddan.github.io/react-spreadsheet/api#CellComponentProps). Is there any alternative to achieve this result?
